Documentation/gettingstarted/policy-creation.rst

```diff
@@ -0,0 +1,161 @@
+
+   # cilium monitor -t policy-verdict
+   ...
+   Policy verdict log: flow 0x63113709 local EP ID 232, remote ID 31028, dst port 80, proto 6, ingress true, action audit, match none, 10.0.0.112 :54134 -> 10.29.50.40:80 tcp SYN
+
+In the above example, we can see that endpoint ``232`` has received traffic
+(``ingress true``) which doesn't match the policy (``action audit match
+none``). The source of this traffic has the identity ``31028``. Let's gather a
+bit more information about what these numbers mean:
+
+.. code-block:: shell-session
+
+   # cilium endpoint list
+   ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                                 IPv6                 IPv4            STATUS
+              ENFORCEMENT        ENFORCEMENT
+   232        Disabled (Audit)   Disabled          16530      k8s:class=deathstar                                                              10.29.50.40     ready
+                                                              k8s:io.cilium.k8s.policy.cluster=default
+                                                              k8s:io.cilium.k8s.policy.serviceaccount=default
+                                                              k8s:io.kubernetes.pod.namespace=default
+                                                              k8s:org=empire
+   ...
+   # cilium identity get 31028
+   ID     LABELS
+   31028  k8s:class=tiefighter
+          k8s:io.cilium.k8s.policy.cluster=default
+          k8s:io.cilium.k8s.policy.serviceaccount=default
+          k8s:io.kubernetes.pod.namespace=default
+          k8s:org=empire
+
+Given the above information, we now know the labels of the target pod, the
+labels of the peer that's attempting to connect, the direction of the traffic
+and the port. In this case, we can see clearly that it's an empire craft
+so once we've determined that we expect this traffic to arrive at the deathstar
+(what could go wrong?), we can form a policy to match the traffic:
+
+.. literalinclude:: ../../examples/minikube/sw_l3_l4_policy.yaml
+
+To apply this L3/L4 policy, run:
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/sw_l3_l4_policy.yaml
+    ciliumnetworkpolicy.cilium.io/rule1 created
+
+Now if we run the landing requests again, we can observe in the monitor output
+that the traffic which was previously audited to be dropped by the policy are
+now reported as allowed:
+
+.. code-block:: shell-session
+
+    $ kubectl exec tiefighter -- curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
+    Ship landed
+
+Executed from the cilium pod:
+
+.. code-block:: shell-session
+
+   # cilium monitor -t policy-verdict
+   Policy verdict log: flow 0xabf3bda6 local EP ID 232, remote ID 31028, dst port 80, proto 6, ingress true, action allow, match L3-L4, 10.0.0.112 :59824 -> 10.0.0.147:80 tcp SYN
+
+Now the policy verdict states that the traffic would be allowed: ``action
+allow``. Success!
```

Comment:
I'd like to end this up a bit less abruptly but I'm running out of creative juice at the moment :-)

Reply:
that's what you get for protecting the empire man.